.github/workflows/compatibility-eap.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
           restore-keys: ${{ runner.os }}-gradle
 
-      - name: Build the plugin
+      - name: Build the plugins
```

Comment:
here as well - 2 jobs is better imo

Reply:
Why. most of the time is building `common`. this way it builds only once.

Reply:
mmm but if the `Tabnine` build breaks you don't get to see if the `SelfHosted` build passes or not.
I don't really care either way, need to work with the CI a bit to get a sense of what's working best.